examples/readintlist.cf

```diff
@@ -0,0 +1,34 @@
+#+begin_src prep
+#@ ```
+#@ printf "one\ntwo\nthree" > /tmp/list.txt
+#@ printf "1\n2\n3"         >> /tmp/list.txt
+#@ printf "1.0\n2.0\n3.0"   >> /tmp/list.txt
+#@ ```
+#+end_src
+bundle agent example_readintlist
+# @brief Example showing function return types
+{
+  vars:
+      "my_list_of_integers"
+        ilist => readintlist( "/tmp/list.txt",      # File to read
+                              "^(\D+)|(\d+[^\n]+)", # Ignore any non integers
+                              "\n",                 # Split on newlines
```

Comment:
Why do you have to provide this regex for non-ints when the function name is readintlist()? Seems like that would be a default.

Reply:
Then again, you are just documenting here, not implementing how it works. So all good, just struck me as curious.

Reply:
I had to supply it because if non-numbers were parsed into the list, the agent would error. So yes, I was just trying to document how to use it with a file that might contain non-numbers. Previously, the examples showed just ignoring hash prefixed lines (which really made sense mostly for slist).